config/environments/development.rb

```diff
@@ -46,7 +46,7 @@
 
   ###################################################################
   # Set the course name here
-  COURSE_NAME         = 'CSC108 Fall 2009: Introduction to Computer Programming'
+  COURSE_NAME         = "CSC108 Fall 2009: Introduction to Computer Programming"
```

Comment:
Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.